backend/src/main/java/atlantbh/restaurants/controllers/dto/LoginRequestDTO.java

```diff
@@ -0,0 +1,23 @@
+package atlantbh.restaurants.controllers.dto;
+
+public class LoginRequestDTO {
+    String email;
```

Comment:
these can be private i guess.

Reply:
Fixed in 92c4f7264b274d7da369717e0ed1478a93d30ee4